js/geo/transform.js

```diff
@@ -375,7 +394,8 @@ Transform.prototype = {
                 y2 !== undefined ? y2 : this.y));
         }
 
-        this._constraining = false;
+        this._unmodified = this._constraining.unmodified;
+        this._constraining = undefined;
```

Comment:
OK, pushed the change. No reason to get annoyed by constructive feedback, this is what PRs are for. :)